### PR TITLE
Kristiyan

### PR DESCRIPTION
**Pull request by Danar435**
Little fixups.

### CODE REVIEW

File: src/problem-analysis/audience.bib [hunk 1]
```diff
@@ -25,3 +25,11 @@ @misc{om_rejseplanen
     note = {Last accessed 11 November 2023},
     url = {https://help.rejseplanen.dk/hc/da/articles/115000127329-Mere-om-Rejseplanen}
 }
+
+@misc{thispersondoesnotexist,
```
Comment (audio-engineer) [inline src/problem-analysis/audience.bib:29]: @Danar435 `@book`, `@article`? https://www.overleaf.com/learn/latex/Bibliography_management_with_bibtex

File: src/problem-analysis/audience.tex [hunk 2]
```diff
@@ -10,8 +10,8 @@ \subsection{Audience}\label{subsec:audience}
 Our concern is about travelling, which covers a wide range of people, as everyone travels one way or another.
 Some examples include travelling to work, to school, family visits, or outings with friends.
 Therefore, we have to narrow it down, as we can't address everyone.
-There are already some solution to that issue, but from our `analysis over existing solutions', we figured that they are
-more fit for single trips, rather than every day travelling.
+There are already some solution to that issue, but from the `Extant solutions and their operational mechanisms' section
```
Comment (audio-engineer) [inline src/problem-analysis/audience.tex]: @Danar435 `but in`?

File: src/problem-analysis/audience.tex [hunk 3]
```diff
@@ -23,14 +23,16 @@ \subsection{Age range}\label{subsec:age-range}
...
 At shorter distances, we would need to exclude planes, trains, and for people younger than 18, cars, making it a matter
-of whether the student should commute by walking or by bike, which are both \unit{CO_{2}} friendly options.
-This makes our solution irrelevant for this age group.
-Things change from secondary education however, as students can freely pick which high school they want to go to.
-This makes them a better candidate for our problem, as they can choose a school farther away from where they live.
+of whether the student should commute by walking or by bike, which are both \unit{CO_{2}} friendly options, and that
+makes our solution irrelevant for this age group.
+Things change from secondary education however, as students can freely pick which high school they want to go to and the
+closest school will be chosen.
+This however makes them a better candidate for our problem, as there are not as many high schools as there are primary,
```
Comment (audio-engineer) [inline src/problem-analysis/audience.tex]: @Danar435 `This, however, makes them better candidates for our problem`

File: src/problem-analysis/audience.tex [hunk 4]
```diff
@@ -49,8 +51,10 @@ \subsection{Persona}\label{subsec:persona}
 To further analyze our age group, we decided to use the \textit{persona} method.
 How it works is that we create fictional people with different issues and look into how our solution could help them
 with their issue.
-We have made 3 personas, where each of them cover different age groups and have distinct characteristics related to the
-main problem.
+We have made 3 personas, with their images generated by GAN AI~\cite{thispersondoesnotexist}, where each of them cover
```
Comment (audio-engineer) [inline src/problem-analysis/audience.tex]: @Danar435 `three`

**Review by audio-engineer — CHANGES_REQUESTED** [review]

File: src/problem-analysis/audience.tex (hunk 3 above)
Comment (Danar435) [reply src/problem-analysis/audience.tex]: Textidote should've caught this but ok

Comment (audio-engineer) [reply src/problem-analysis/audience.tex]: Did you check whether TeXtidote is linting your files as well? Check the `README.md` again

File: src/problem-analysis/audience.bib (hunk 1 above)
Comment (Danar435) [reply src/problem-analysis/audience.bib:29]: Folk anbefaler @misc til websider. financial-awareness.bib har @online, men jeg tror ikke vi bruger BibLaTeX.
https://tex.stackexchange.com/questions/3587/how-can-i-use-bibtex-to-cite-a-web-page

**Review by audio-engineer — APPROVED** [review]

**Review by TexWO — APPROVED** [review]

**Pull request merged**